tools/SourceKit/lib/SwiftLang/SwiftLangSupport.h

```diff
@@ -270,35 +271,35 @@ struct SwiftStatistics {
 };
 
 class SwiftLangSupport : public LangSupport {
-  SourceKit::Context &SKCtx;
+  std::weak_ptr<NotificationCenter> NotificationCtr;
```

Comment:
Why is this weak_ptr, `NotificationCenter` doesn't seem to be forming a cycle with `SwiftLangSupport`.

Reply:
Ah, good point.